dnsaas/api/v2/serializers.py

```diff
@@ -115,6 +126,27 @@ def _clean_txt_content(self, record_type, attrs):
         if record_type == 'TXT':
             attrs['content'] = attrs['content'].replace('\\', '')
 
+    def _validate_service(self, attrs):
+        if (
+            # enforce service required for patch updates
```

Comment:
so if service field is required and we're not allowing for partial updates (PATCH), why is this validator necessary?